samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -518,7 +518,7 @@ timer_preview_not_available = Timer does not work in Preview
 fin_accepted_instruction=<b style="color:#860000;">Accepted characters</b>: numbers, decimal point markers, sign indicators (-), spaces (e.g., as thousands separator, 5 000), "E" or "e" (used in scientific notation). <b>NOTE:</b> For scientific notation, a period MUST be used as the decimal point marker.
 fin_complex_note=Complex numbers should be in the form {a + bi} where "a" and "b" need to have explicitly stated values.
 fin_complex_example=For example: {1+1i} is valid whereas {1+i} is not. {0+9i} is valid whereas {9i} is not.
-fin_invalid_characters_error=Please use only accepted characters within numeric response fields.
+fin_invalid_characters_error=Please use only accepted characters within numeric response fields. Accepted inputs are: Reals like 9.3. Complex numbers like {9+10i}. Do not forget the brackets. Scientific notation like 6.32e10 or 6.32E10.
```

Comment:
The phrasing here is a little awkward. I would suggest the following:

> Please only use allowed characters within numeric response fields. Acceptable inputs are real numbers like "9.3", complex numbers like "{9+10i}" (with the required braces), and scientific notation like "6.32e10" or "6.32E10".

Reply:
Makes sense to me @bjones86, thanks for the suggestion. I've updated the text.